src/nvim/options.lua

```diff
@@ -1599,7 +1599,7 @@ return {
       deny_duplicates=true,
       alloced=true,
       varname='p_nf',
-      defaults={if_true={vi="octal,hex", vim="hex"}}
+      defaults={if_true={vi="bin,octal,hex", vim="bin,hex"}}
```

Comment:
Though `'nrformats'` is marked as “not in vi”, as well as `CTRL-A` and `CTRL-X`, so this does not matter.